GVFS/GVFS/CommandLine/UpgradeVerb.cs

```diff
@@ -94,6 +94,15 @@ private bool TryInitializeUpgrader(out string error)
                 error = null;
                 if (this.upgrader == null)
                 {
+                    string createDirectoryError;
+                    if (!this.fileSystem.TryCreateDirectoryWithAdminAndUserModifyPermissions(ProductUpgraderInfo.GetLogDirectoryPath(), out createDirectoryError))

```

Comment:
Can this be run from a non-admin command prompt?

Reply:
> Can this be run from a non-admin command prompt?

Under normal circumstances yes.  

GVFS.Service will create the `UpgraderLogs` directory with the correct ACLs when it starts up.

However, if something during an install\upgrade went wrong the the service failed to start the `UpgraderLogs` directory would not be created by the service and it's possible that a non-elevated user would be unable to create the directory.

Reply:
I will add a comment to the code with these details.